doc/versionHistory.md

```diff
@@ -1,5 +1,10 @@
 # Version History
 
+0.2.2
+
+- TCP Server can receive telemetry from TCP Client.
+- TCP Server can send telemetry to TCP Client.
+
```

Comment:
Please also say this version moves the attributes of `firstPacket` and `lastSequenceId` as from **tcpServerBase** to **tcpServerCmd** with great details. Thanks!

Reply:
Done.